### PR TITLE
[bare] Add header and footer

### PR DESCRIPTION
**Pull request by leticiansantos**
- Create header and footer structure considering visual identity for the brand

Notes:
- Missing to add the discord icon in the footer, since it's available only in the new fontawesome
- Also using some copy/paste css. maybe not the best elements naming

### CODE REVIEW

File: index.html
Comment (leticiansantos) [inline index.html:1]: Maybe you can remove this one.

File: public/vite.svg
Comment (leticiansantos) [inline public/vite.svg:1]: We can change this icon for a mentoria bare one

File: src/index.css
Comment (leticiansantos) [inline src/index.css:1]: Maybe we can delete this

**Review by leticiansantos — COMMENTED** [review]
Some nit comments. But since it's MVP not a problem.

LGTM! Thanks for doing that!

**Review by guilhermemdasilva — APPROVED** [review]
LGTM!

**Pull request merged**